tests/NoMissingTypeExposeTest.elm

```diff
@@ -457,6 +457,228 @@ type Mood
     = Happy Happiness
 
 
+type Happiness
+    = Ecstatic
+    | FineIGuess
+    | Unhappy
+"""
+                    ]
+    , test "reports when an exposed typed type alias uses a private type" <|
+        \() ->
+            """
+module Happiness exposing (Mood)
+
+
+type alias Mood =
+    Maybe Happiness
+
+
+type Happiness
+    = Ecstatic
+    | FineIGuess
+    | Unhappy
+"""
+                |> Review.Test.run rule
+                |> Review.Test.expectErrors
+                    [ Review.Test.error
+                        { message = "Private type `Happiness` used by exposed function"
+                        , details =
+                            [ "Type `Happiness` is not exposed but is used by an exposed function."
```

Comment:
I think we can just remove this line. The `message` already explains it, and the following line already explains the problem well.